scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestController.java

```diff
@@ -35,7 +35,12 @@
 	public ResponseEntity<ApplicationScalingSchedules> getAllSchedules(@PathVariable String app_id) {
 		logger.info("Get All schedules for application: " + app_id);
 		ApplicationScalingSchedules savedApplicationSchedules = scheduleManager.getAllSchedules(app_id);
-
+		
+		// Any exception in getting the schedules
```

Comment:
We get an exception in that case. For instance, database connection error.
We will remove check for the ValidationErrorResult in case of get if you think it is not justified.